doc/website/concepts/architecture.md

```diff
@@ -1 +1,65 @@
 # Architecture
+
+This article provides an overview of the Eclipse iceoryx architecture and explains the rationale for it.
+
+## Software layers
+
+The main packages which Eclipse iceoryx is composed of are depicted below.
+
+![Software layers](../images/iceoryx_software_layers_v2_0_0.svg)
+
+The next sections briefly describe the components and their libraries one by one.
+
+## Components and libraries
+
+The different libraries and their namespaces are depicted below.
+
+![Component diagram](../images/iceoryx_components_diagram_v2_0_0.svg)
+
+### iceoryx hoofs
+
+Handy Objects Optimized For Safety (hoofs) is a library and contains various building blocks like fixed size containers,
+concurrency classes and modern, next-gen C++ constructs from upcoming C++ standard releases.
+
+For more information about the components, refer to its [detailed description](../advanced/iceoryx_hoofs.md).
```

Comment:
When I follow up this link I do not get a detailed description I only come to some simple stub document. Shouldn't we adjust the link to this document: https://github.com/eclipse-iceoryx/iceoryx/blob/ada657a42e8a039e685f9807b11c01fee378678c/iceoryx_hoofs/README.md

Reply:
Ok, to unblock this discussion. Let's just use the full http link for now..